### PR TITLE
Creating aggregation driver for resource Field of microservice Satell…

### PR DESCRIPTION
**Pull request by DouglasVinicius**
…ite.

### CODE REVIEW

**Comment by DouglasVinicius** [comment]
[[BillingAg] Create aggregation driver for microservice Satellite with resource Field](https://trello.com/c/ZW9OAKMW/995-billingag-create-aggregation-driver-for-microservice-satellite-with-resource-field)

**Review by rgdafonso — APPROVED** [review]
Nice! Good job!

File: src/main/java/com/leaf/billing/SatelliteFieldAggregatorDriver.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+public class SatelliteFieldAggregatorDriver {
+    /**
+     * This spark job will aggregate the parquet files into a data frame
+     *
+     * @param args job parameters
+     */
+    public static void main(String[] args) throws IOException {
+        JobProperties jobProperties = PropertyBuilder.build(args);
+        try (SparkSession sparkSession = SparkSession.builder()
+                .appName("BillingAggregator4")
```
[the remark this answers is not in the document]
Comment (DouglasVinicius) [reply src/main/java/com/leaf/billing/SatelliteFieldAggregatorDriver.java:31]: I kept the pattern of others "driver" application, but we can review these names.
Thanks for the suggestion Roger! :D

**Review by ricardosantacattarina — APPROVED** [review]
Looks good!

File: src/main/java/com/leaf/billing/SatelliteFieldAggregatorDriver.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (fabyan-alexander) [reply src/main/java/com/leaf/billing/SatelliteFieldAggregatorDriver.java:31]: Can we use the class name on the appName?

[the remark this answers is not in the document]
Comment (DouglasVinicius) [reply src/main/java/com/leaf/billing/SatelliteFieldAggregatorDriver.java]: Thanks guys, here is the commit that implemented this modification: https://github.com/Leaf-Agriculture/billing-data-aggregation-job/pull/8/commits/fed2387c8daf560d5e857a867c03acf554b4167b

**Review by rgdafonso — APPROVED** [review]
Good job on fixing the other appNames

**Review by fabyan-alexander — APPROVED** [review]
Nice work!

**Pull request merged**